src/hooks/serverStateHooks/useGetArtworkComments.ts

```diff
@@ -0,0 +1,90 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+import { useEffect, useMemo, useRef, useState } from 'react';
+
+import getArtworkComments from '@/apis/artwork/getArtworkComments';
+import { ARTWORK } from '@/constants/API';
+import { ArtworkComment } from '@/types';
+
+interface UseGetArtworkComments {
+  postId: number;
+  size?: number;
+  skip: number;
+}
+
+const useGetArtworkComments = ({
+  postId,
+  size = 5,
+  skip = 0,
+}: UseGetArtworkComments) => {
+  const {
+    data: commentData,
+    fetchNextPage,
+    hasNextPage,
+    isFetching,
+    isFetchingNextPage,
+  } = useInfiniteQuery({
+    queryKey: [ARTWORK.artworkPostComments, postId, { size, skip }],
```

Comment:
```suggestion
    queryKey: [ARTWORK.artworkPostComments, postId, { size }],
```

TanStack Query가 `useInfiniteQuery`에서 `pageParam`을 자동으로 관리하고 있는데, `queryKey`에도 `skip`이 포함해야 하는 이유가 무엇인지 궁금합니당

Reply:
useInfiniteQuery 내부적으로 관리하는 것은 몰랐습니다! 그래서 추후 skip 이 key에 포함되어야 원하는 대로 적절한 캐싱이 이루어진다고 생각했습니다..! 해당 부분은 삭제하면 될까요?

Reply:
useInfiniteQuery에서 내부적으로 params를 관리하는 것은 제가 미처 모르고 있는 부분이었습니다. 해당 params는 삭제하면 될까요?

Reply:
아 그러셨군요. 네네 삭제하셔야 상우님이 생각하신 캐싱대로 작동할 듯합니다!